src/main/java/com/mraof/minestuck/network/LotusFlowerPacket.java

```diff
@@ -5,7 +5,8 @@
 import net.minecraft.network.FriendlyByteBuf;
 import net.minecraft.world.entity.Entity;
 
-public class LotusFlowerPacket implements PlayToClientPacket
+public class
+LotusFlowerPacket implements PlayToClientPacket
```

Comment:
Same thing as what I commented in GristTypes

Reply:
u think this was resolved? i did the ctrl thingy multiple times